src/databases/CMakeLists.txt

```diff
@@ -376,11 +379,7 @@ endif()
 IF(WIN32)
```

Comment:
Can this code in the IF block be moved out, since both halves do the same thing?

Reply:
Good catch!